lnurl/core.py

```diff
@@ -52,3 +54,53 @@ def handle(
         return LnurlAuthResponse(callback=lnurl.url, k1=lnurl.url.query_params["k1"])
 
     return get(lnurl.url, response_class=response_class, verify=verify)
+
+
+def execute(bech32_or_address: str, value: str) -> LnurlResponseModel:
+    try:
+        res = handle(bech32_or_address)
+    except Exception as exc:
+        raise LnurlResponseException(str(exc))
+
+    if isinstance(res, LnurlPayResponse) and res.tag == "payRequest":
```

Comment:
nitpick: no the '==' comparison first:

```suggestion
    if res.tag == "payRequest" and isinstance(res, LnurlPayResponse):
```

same below

Reply:
this throw and linting issue because it cannot find the tag in the the repsonse